internal/updater/self-updater.go

```diff
@@ -102,8 +103,10 @@ func (u *SelfUpdater) checkSelfUpdate() <-chan bool {
 			return
 		}
 
-		ch <- u.latestVersion.Version != u.CurrentVersion &&
+		// Only offer update if remote version is newer than current version
+		shouldUpdate := remote.IsVersionSmaller(u.CurrentVersion, u.latestVersion.Version) &&
```

Comment:
The rational of comparing exact version here is for the situation to rollback to a previous versions. This happens sometimes when a new version of command launcher is released, but some packages are not compatible to it. As a command launcher admin, having a rollback options is always a good thing. 
So a typical rollback procedure is simply make the `self_update_latest_version_url` return an older version.
With this change, the rollback has to go through a new release process with a newer version.